Create default ProjectType/Stack/Template as part of setup (#1250)
flowforge/flowforge: docs/install/local/stacks.md

    is useful when running on a memory constrained device such as a Raspberry Pi. Recommended minimum `256`.
 - `nodered` - the version number of Node-RED to use. This should match the value used in the steps following.

The FlowForge installer will create a default stack using the latest stable
release of Node-RED.

> The stacks are stored under `/opt/flowforge/var/stacks` or `c:\opt\flowforge\var\stacks` on Windows.

Reviewer: ```suggestion
The stacks are stored under `/opt/flowforge/var/stacks` or `c:\flowforge\var\stacks` on Windows.
```

Docs suggest install to `c:\flowforge`

### Creating a Stack

When a new version of Node-RED is released, it can be added to your FlowForge
platform by creating a new stack.

For a local install there are two steps required:

1. Install a new Node-RED version

   In the FlowForge Home directory, run the provided install script. You
   must provide the full Node-RED version number, eg `3.0.2`, or use `latest` to install the most recent stable version.

   Linux/Mac:
   ```
   cd /opt/flowforge
   ./bin/ff-install-stack.sh 3.0.2
   ```

   Windows
   ```

>    cd c:\opt\flowforge

Reviewer: ```suggestion
   cd c:\flowforge
```

   bin\ff-install-stack.bat 3.0.2
   ```

2. Creating the Stack

   Log into the FlowForge platform as an administrator. Navigate to the
   Admin Settings -> Stacks section.

   If this new stack should be offered as a direct upgrade of an existing stack, select the 'Create new version' option from the dropdown menu
   of the stack you want to replace. Projects that use the old stack
   will offer the new stack as a one-click upgrade option.

   Alternatively, click 'Create stack' to create an entirely new stack.

   When prompted for the Node-RED version, provide the exact version that was installed. For example, if you ran the script with `latest` and it resulted in `3.0.2` being installed, you should enter `3.0.2`. This must

>    match the directory name created under `/opt/flowforge/var/stacks`.

Reviewer: ```suggestion
   match the directory name created in your `stacks` directory.
```

Suggestion: make this system agnostic?
